language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/FieldAccessCompletionResolver.java

```diff
@@ -82,14 +83,12 @@ public FieldAccessCompletionResolver(PositionedOperationContext context) {
 
     @Override
     public Optional<TypeSymbol> transform(SimpleNameReferenceNode node) {
-        Optional<Symbol> symbol = this.getSymbolByName(context.visibleSymbols(context.getCursorPosition()),
-                node.name().text());
-
-        if (symbol.isEmpty()) {
-            return Optional.empty();
-        }
+        return getTypeDescriptor(node.name().text());
+    }
 
-        return SymbolUtil.getTypeDescriptor(symbol.get());
+    @Override
+    public Optional<TypeSymbol> transform(ErrorConstructorExpressionNode node) {
+        return getTypeDescriptor(node.errorKeyword().text());
```

Comment:
What if we use the `typeOf` API in semantic model?

Reply:
@nadeeshaan Also works. Shall I make the change in both `SimpleNameReferenceNode` and `ErrorConstructorExpressionNode`?